pkg/kapp/resourcesmisc/custom_waiting_resource.go

```diff
@@ -96,6 +98,12 @@ func (s CustomWaitingResource) IsDoneApplying() DoneApplyState {
 					continue
 				}
 				if condMatcher.Success {
+					if condMatcher.SupportsUnblockingChanges {
```

Comment:
seems like adding this here is not enough if we would want to support same feature from ytt wait rules as well. (seems a bit awkward if we do not).

Reply:
Done.